tools/analysis/maian/analysis.md

```diff
@@ -0,0 +1,67 @@
+# MAIAN
+
+[MAIAN](https://github.com/ivicanikolicsg/MAIAN) is a tool specifically designed to find security vulnerabilities
+in Smart Contracts that operate on Ethereum and other blockchain platforms. MAIAN main aim is to detect three
+types of problematic contracts:
+- **suicidal contract**: one that can be forced to self-destruct by any external user, not just their owner.
+- **prodigal contract**: one that can inadvertently leak funds to arbitrary addresses due to improper handling of
+  Ether transfers.
+- **greedy contract**: one that locks funds indefinitely, making them irretrievable.
+  Its key features are:
+- Capability of analyzing both Solidity source code and bytecode.
+- Automated analysis with detailed reporting of vulnerabilities.
+- Graphical representations of control flow to aid in the understanding of contract behavior.
+- Integrated support for deployment analysis.
+- Easily integrates with continuous integration pipelines for ongoing security assessment.
+  The tool is especially effective at finding specific types of security flaws that involve the misuse of funds within contracts
+
+## Installation:
+Due to changes in web3.py library introduced in version 3.5.5V (including), an attempt to start MAIAN, using instructions from
+[github repo](https://github.com/ivicanikolicsg/MAIAN), ends with an error prompt that may be found [here](example_build_execution_error.md)
+Further discussion regarding this error may be found [here](https://github.com/ethereum/web3.py/issues/879)
+Alternative ways to run MAIAN are:
+1) Running with manual downgrade od Python to v2 and web3.py to 3.5.4V
+2) Running inside the Docker container:
+```Shell
+docker pull cryptomental/maian-augur-ci
+docker run -it cryptomental/maian-augur-ci
+cd MAIAN/tool && python2 maian.py -s example_contracts/example_suicidal.sol KAI -c 0
+```
+> **NOTE**: There is no `solc-select preinstalled` in the docker image. In order to change the version run:
+> ```shell
+> wget https://github.com/ethereum/solidity/releases/download/{your_[docker-compose.yaml](docker-compose.yaml)
+[analysis.md](analysis.md)
+[signing_transactions.md](signing_transactions.md)version}/solc-static-linux && mv solc-static-linux /usr/bin/solc
+> ```
+> where `{your_version}` is desired solidity compiler version, ie. v0.8.24.
+
+## Tool analysis:
+### Attempt of replacing Ethereum's JSON RPC to Hedera's JSON RPC relay:
+As of April 2024, MAIAN across the last six years was not in active development. This state leads the tool to be
+out of date and lacks support for newer features. This leads to multiple challenges, most impactful ones
+are the following:
+- **Differences in RPC Methods Ethereum's JSON RPC API**: Hedera does not support `eth_sendTransaction`.
```

Comment:
I think you should point out that this is similar to other notable RPCs that also don't support this endpoint do to the desire to not handle keys for users. As written it sounds like Hedera is incapable of supporting and that it's odd in this case, whereas it's a security choice made similar to other RPCs. 

You can also link to the `tools/analysis/maian/signing_transactions.md` README where you've done a great job of calling the differences out
```suggestion
- **Differences in RPC Methods Ethereum's JSON RPC API**: Hedera does not support `eth_sendTransaction`.
```